ObjectDetectionCameraStream.ipynb

```diff
@@ -120,20 +112,10 @@
    "source": [
     "# AI prediction loop\n",
     "# Press 'x' or 'q' to stop\n",
-    "with mytools.Display(\"AI Camera\") as display, \\\n",
-    "     mytools.open_video_stream(camera_id) as stream:\n",
-    "    \n",
-    "    for res in model.predict_batch(mytools.video_source(stream)):\n",
-    "        display.show(res.image_overlay)"
+    "with mytools.Display(\"AI Camera\") as display:    \n",
+    "    for res in mytools.predict_stream(model, camera_id):\n",
+    "        display.show(res)"
```

Comment:
@vlad-nn does this code work today? I am asking because current default backend is PIL and display while understands PIL results may not be able to show videos.